investd/sources/__init__.py

```diff
@@ -15,12 +16,10 @@
 ]
 
 
-def ingest_sources_as_df() -> pd.DataFrame:
-    df_tx = pd.DataFrame(columns=Transaction.__dataclass_fields__.keys())
-    for source_class in sources:
-        source = source_class()
-        df_tx = pd.concat(
-            [df_tx, pd.DataFrame([tx for tx in source.load_transactions()])]
-        )
+def ingest_all() -> pd.DataFrame:
+    df_tx = pd.DataFrame(
```

Comment:
for some reason ingest_all gives me an error:

      raise ValueError("len(arrays) must match len(columns)")
       E ValueError: len(arrays) must match len(columns)

when I added list() wrapper to de-lazify both columns and data, it works

I think it happens when sources are empty